content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -538,17 +538,15 @@ The following query on the unsupported field will **not** work:
 await prisma.$queryRaw`SELECT location FROM Country;`
 ```
 
-Instead, cast `Unsupported` fields to any supported Prisma type, **if your `Unsupported` column supports the cast**.
+Instead, cast `Unsupported` fields to any supported Prisma Client type, **if your `Unsupported` column supports the cast**.
 
 The most common type you may want to cast your `Unsupported` column to is `String`. For example, on PostgreSQL, this would map to the `text` type:
 
 ```tsx
 await prisma.$queryRaw`SELECT location::text FROM Country;`
 ```
 
-The database will thus provide a `String` representation of your data which Prisma supports.
-
-For details of supported Prisma types, see the [Prisma data connector](/orm/overview) for the relevant database.
```

Comment:
Having a link to an overview of the types seems useful here. Removed because the link target is probably not that useful right now? Can we not just find a better one?

Reply:
Similar links link to https://www.prisma.io/docs/orm/overview/databases I think, and use "Prisma connector overview" for example as the link text.

Reply:
Do you want me to add a line on 551 that says: 

For details of supported Prisma types, see the [Prisma connector overview](/orm/overview/databases) for the relevant database.

Reply:
Yes, I think that sentence was useful before 👍